akka-persistence/src/test/scala/akka/persistence/SnapshotRecoveryLocalStoreSpec.scala

```diff
@@ -22,9 +22,10 @@ object SnapshotRecoveryLocalStoreSpec {
   }
 
   class LoadSnapshotTestPersistentActor(name: String, probe: ActorRef) extends NamedPersistentActor(name)
-    with TurnOffRecoverOnStart
     with ActorLogging {
 
+    override def recovery = Recovery(toSequenceNr = 0)
```

Comment:
now I'm confused. Does this introduce a changed behavior or not? Is it the inmem journal that doesn't respect the toSeqnr correctly that is set via `criteria.limit(toSequenceNr)`?